adafruit_fanservice.py

```diff
@@ -0,0 +1,131 @@
+import sys
+import os
+import subprocess
+
+group = 'ADAFRUIT'
+
+class ColorPrint:
+    @staticmethod
+    def print_fail(message, system=None, end = '\n'):
+        if system is None:
+            system = group
+        sys.stdout.write('\x1b[1;31m\x1b[40m' + system.strip() + '\x1b[0m ' + message.strip() + end)
+
+    @staticmethod
+    def print_pass(message, system=None, end = '\n'):
+        if system is None:
+            system = group
+        sys.stdout.write('\x1b[1;32m\x1b[40m' + group.strip() + '\x1b[0m ' + message.strip() + end)
+
+    @staticmethod
+    def print_warn(message, system=None, end = '\n'):
+        if system is None:
+            system = group
+        sys.stdout.write('\x1b[1;33m\x1b[40m' + group.strip() + '\x1b[0m ' + message.strip() + end)
+
+    @staticmethod
+    def print_info(message, system=None, end = '\n'):
+        if system is None:
+            system = group
+        sys.stdout.write('\x1b[1;34m\x1b[40m' + group.strip() + '\x1b[0m ' + message.strip() + end)
+
+    @staticmethod
+    def print_bold(message, system=None, end = '\n'):
+        if system is None:
+            system = group
+        sys.stdout.write('\x1b[1;37m\x1b[40m' + group.strip() + '\x1b[0m ' + message.strip() + end)
+
+def info(system, message):
+    group = system
+    ColorPrint.print_pass(message)
+
+def bail(message = None):
+    if message is None:
+        ColorPrint.print_fail("Exiting due to error")
+    else:
+        ColorPrint.print_fail("Exiting due to error: {message}".format(message=message))
+    sys.exit(1)
+
+def run_command(cmd, suppress_message = False):
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE,
+                          stderr=subprocess.PIPE)
+    r = proc.wait()
+    out = proc.stdout.read()
+    err = proc.stderr.read()
+    if r == 0:
+        return True
+    else:
+        if not suppress_message:
+            ColorPrint.print_fail(err.decode("utf-8"))
+        return False
+
+def main():
+    os.system('clear')
+    print("This script will install Adafruit")
```

Comment:
use """ to make one block https://stackoverflow.com/questions/34980251/how-to-print-multiple-lines-of-text-with-python